google/cloud/sql/connector/connector.py

```diff
@@ -171,7 +172,11 @@ def __init__(
         if isinstance(ip_type, str):
             ip_type = IPTypes._from_str(ip_type)
         self._ip_type = ip_type
-        self._universe_domain = universe_domain
+        # check for universe domain arg and then env var
+        if universe_domain:
+            self._universe_domain = universe_domain
+        else:
+            self._universe_domain = os.environ.get("GOOGLE_CLOUD_UNIVERSE_DOMAIN")  # type: ignore
```

Comment:
need to have `# type: ignore` here as os.environ.get has weird type hinting

was causing mypy check to complain during lint:

```
google/cloud/sql/connector/connector.py:179: error: Incompatible types in assignment (expression has type "Optional[str]", variable has type "str")  [assignment]
```